hbase-server/src/test/java/org/apache/hadoop/hbase/master/balancer/TestDefaultLoadBalancer.java

```diff
@@ -58,7 +57,7 @@
 
   private static final Logger LOG = LoggerFactory.getLogger(TestDefaultLoadBalancer.class);
 
-  private static LoadBalancer loadBalancer;
+  private static SimpleLoadBalancer loadBalancer;
```

Comment:
This test is for SimpleLoadBalancer? Rename TestDefaultLoadBalancer to TestSimpleLoadBalancer?